### PR TITLE
SDIT-1126 prison-api startup takes a lot of cpu

### PR DESCRIPTION
**Pull request by steverendell**
Set one pod per node (5) in prod and tune memory and cpu

### CODE REVIEW

File: helm_deploy/prison-api/values.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 generic-service:
   nameOverride: prison-api
   productId: DPS060
-  replicaCount: 6
+  replicaCount: 5 # One per node
```
Comment (steverendell) [inline helm_deploy/prison-api/values.yaml:6]: The number of nodes available to run pods on has increased from 3 to 5

File: helm_deploy/prison-api/values.yaml [hunk 2]
```diff
@@ -27,14 +27,14 @@ generic-service:
 
   resources:
     requests:
-      cpu: 100m
-      memory: 640Mi
+      cpu: 1000m
+      memory: 2048Mi
```
Comment (steverendell) [inline helm_deploy/prison-api/values.yaml:31]: Make these values more realistic (requests is minimum required)

File: helm_deploy/values-dev.yaml [hunk 3]
```diff
@@ -6,6 +6,14 @@ generic-service:
...
     enabled: true
 
+  resources:
+    requests:
+      cpu: 100m
+      memory: 1024Mi
+    limits:
+      cpu: 2000m
+      memory: 2048Mi
+
```
Comment (steverendell) [inline helm_deploy/values-dev.yaml:16]: Convention is for values.yaml to contain production values - so smaller limits go in the env-specific config

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**